src/scrabble.js

```diff
@@ -2,31 +2,80 @@ const { letters } = require('./letter.js')
 
 const scrabble = (word) => {
   let output = 0
-  const scrabbleArr = []
+  let mulitplier = 1
+  // Arrays to test for valid input
+  const checkerArr = []
+  const checkerArr2 = []
 
+  // Checking for any number in input
   function checkString(string) {
     return /^[0-9]*$/.test(string)
   }
 
   const stringContainsNum = checkString(word)
 
-  if (typeof word === 'string' && stringContainsNum === false) {
+  // Multiple checks for valid input
+  if (
+    typeof word === 'string' &&
+    stringContainsNum === false &&
+    /^[a-zA-Z{}[\]]*$/.test(word)
```

Comment:
Looks good! I think I've just seen Nazar use sth similar in his code (.test() method) - how does it work?

Reply:
It's new knowledge to me! 
Here's the MDN definition: 
The test() method of RegEx instances executes a search with this regular expression for a match between a regular expression and a specified string. Returns true if there is a match; false otherwise.